tests/test-runner/bin/zts-report.py.in

```diff
@@ -263,6 +263,8 @@ elif sys.platform.startswith('linux'):
         'cli_root/zpool_expand/zpool_expand_001_pos': ['FAIL', known_reason],
         'cli_root/zpool_expand/zpool_expand_005_pos': ['FAIL', known_reason],
         'cli_root/zpool_reopen/zpool_reopen_003_pos': ['FAIL', known_reason],
+        'limits/filesystem_limit': ['SKIP', known_reason],
+        'limits/snapshot_limit': ['SKIP', known_reason],
```

Comment:
Since the test are now self-skipping on unsupported kernels, do we need to list it here?

Reply:
Yes

Reply:
Yes, otherwise the skipped tests are logged as unexpected results.